src/main/resources/openapi/common-schemas.yaml

```diff
@@ -389,6 +389,12 @@ components:
         required:
           type: boolean
           nullable: true
+        autoFillUser:
```

Comment:
Maybe its more readeable / future-proof to make an autoFill field with an enum (user and date for now, maybe more in the future). Also technically you could have both autoFills enabled now.

Reply:
Hm I see they are for date and text fields so different use-cases, ignore my comment